odoo_backup_sh/models/odoo_backup_sh.py

```diff
@@ -63,7 +63,9 @@ class BackupConfig(models.Model):
     unlimited_time_frame = fields.Char(default="hour")
     common_rotation = fields.Selection(selection=ROTATION_OPTIONS, default='unlimited')
     max_backups = fields.Integer(readonly=True, compute=lambda self: 0)
-    backup_simulation = fields.Boolean(string="Demo Backup Simulation", default=False)
+    backup_simulation = fields.Boolean(string="Demo Backup Simulation", default=False,
+                                       help="if the setting is enabled then new backups will not be sent to the "
+                                            "storage server")
```

Comment:
You don't mention here this settings work only for  current database. But I'd prefer to make it work for all databases instead